frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -447,3 +447,94 @@ The list of shared modules planned for deprecation:
+
+```json
+  {
+    "type": "console.navigation/section",
+    "properties": {
+      "id": "admin-demo-section",
+      "perspective": "admin",
+      "name": "%plugin__console-hello-world~Hello World%"
+    }
+  }
+```
+
+Then, run `yarn i18n` to update the JSON files in the `locales` folder of the plugin console after adding or updating strings.
+
+### Step 2: Request a Phrase Translation Memory System (TMS) user account
+
+- Request an account from the localization team for the console plugin project.
+- Create a Project Template to include the supported language in the Phrase TMS portal. Refer to the [Phrase Project Templates ](https://support.phrase.com/hc/en-us/articles/5709647439772-Project-Templates-TMS) on how to create/update the Project Template. You must have Phrase project owner permissions to perform this task. The localization team can help with creating the Project Template and setup.
+
+### Step 3: Create utility scripts to automate i18n-related tasks 
+
+Create scripts for uploading and downloading the i18n JSON files to/from the Phrase portal. See the [console](https://github.com/openshift/console/tree/master/frontend/i18n-scripts) repository or [Advanced Cluster Management (ACM) console plugin](https://github.com/stolostron/console/tree/main/frontend/i18n-scripts) repository for similar scripts.
+
+### Step 4: Upload to Phrase portal
+
+- Install the unofficial Memsource CLI client. This client is a command-line tool designed for interacting with the Phrase portal. See the following link for details: [The unofficial Memsource CLI client](https://github.com/unofficial-memsource/memsource-cli-client#pip-install) 
+
+- Configure the Memsource client using the account credentials. Create a file named ~/.memsourcerc in any location, for example, /Users/.../, and paste the following content into the file:
+
+```
+export MEMSOURCE_URL="https://cloud.memsource.com/web"
+export MEMSOURCE_USERNAME=username
+export MEMSOURCE_PASSWORD=password
+export MEMSOURCE_TOKEN=$(memsource auth login --user-name $MEMSOURCE_USERNAME --password "${MEMSOURCE_PASSWORD}" -c token -f value)
+```
+
+See the following link for details:[The RHEL instructions for MacOS](https://github.com/unofficial-memsource/memsource-cli-client#configuration-red-hat-enterprise-linux-derivatives)
+
+- Change directory to the root of the console plugin, and then run the upload script created earlier in step 3, for example: `yarn memsource-upload -v PLUGIN_VERSION_NUMBER`. Take note of the generated `PROJECT_ID`.
+
+- Notify the localization team of the upload and wait for a reply from them on when the translated strings are ready for download. 
+
+### Step 5: Download from Phrase portal
+
+- Use the `PROJECT_ID` generated during the upload task, or visit the [Phrase TMS](https://cloud.memsource.com) portal. Find the URL of the previously uploaded project, then copy the `PROJECT_ID` from the URL path following `../show/<PROJECT_ID>`
```

Comment:
nit: I feel like these sub-steps should be numbered, since they should occur in a correct order.